# Statistical Analysis of High-Throughput Genomic and Transcriptomic Data 
Fall/Herbst-semester 2017

## Lectures
Mondays 9.00-9.45 (Y27-H-46), 10.00-10.45 (Y11-J-05)

## Exercises
Monday 11.00-11.45 (Y11-J-05)

## Lecturers

[Ms. Gosia Nowicka](https://www.researchgate.net/profile/Malgorzata_Nowicka), finished PhD student, [IMLS](http://www.imls.uzh.ch/index.html), UZH  

[Dr. Hubert Rehrauer](http://www.fgcz.ch/the-center/people/rehrauer.html), Group Leader of Genome Informatics at [FGCZ](http://www.fgcz.ch/)  

[Prof. Dr. Mark Robinson](http://www.imls.uzh.ch/research/robinson.html), Associate Professor of Statistical Genomics, [IMLS](http://www.imls.uzh.ch/index.html), UZH  

[Dr. Charlotte Soneson](http://csoneson.github.io/), Postdoctoral Associate, [IMLS](http://www.imls.uzh.ch/index.html), UZH  


## Schedule

| Date  | Lecturer | Topic | JC1 | JC2 
| --- | --- | --- | --- | --- | 
| 18.09.2017  | Mark  | admin, mol. biology basics, R markdown | | | 
| 25.09.2017  | Hubert  | NGS intro; exploratory data analysis | | | 
| 02.10.2017  | Mark + Hubert  | interactive technology session  | | | 
| 09.10.2017  | Hubert  | mapping  | | | 
| 16.10.2017  | Mark  | limma 1   | | | 
| 23.10.2017  | Mark  | limma 2  | | | 
| 30.10.2017  | Hubert  | RNA-seq quantification   | [Assessment of batch-correction methods for scRNA-seq data with a new test metric](https://www.biorxiv.org/content/early/2017/10/09/200345) (EC) |[Assessing intratumor heterogeneity and tracking longitudinal and spatial clonal evolutionary history by next-generation sequencing](http://dx.doi.org/10.1073/pnas.1522203113) (SS) | 
| 06.11.2017  | Mark  | edgeR+friends 1 | [Why Most Published Research Findings Are False](http://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.0020124); [Is most published research really false?](https://www.biorxiv.org/content/early/2016/04/27/050575) (PM, SS) | [Gene-level differential analysis at transcript-level resolution](https://doi.org/10.1101/190199) (CL) | 
| 13.11.2017  | Charlotte  | hands-on session #1: RNA-seq  | X | X | 
| 20.11.2017  | Mark  | edgeR+friends 2  | [High Dimensional Classification with combined Adaptive Sparse PLS and Logistic Regression-link](https://www.ncbi.nlm.nih.gov/pubmed/28968879) (TF, YY) | [BSmooth: from whole genome bisulfite sequencing reads to differentially methylated regions](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2012-13-10-r83) (SO) | 
| 27.11.2017  | Hubert  | classification  | [Bayesian approach to single-cell differential expression analysis](http://www.nature.com/nmeth/journal/v11/n7/abs/nmeth.2967.html) (UJ) | [Guidance for RNA-seq co-expression network construction and analysis: safety in numbers](https://www.ncbi.nlm.nih.gov/pubmed/25717192) (CS) | 
| 04.12.2017  | Mark  | single-cell  | [Removal of batch effects using distribution-matching residual networks](https://academic.oup.com/bioinformatics/article/33/16/2539/3611270/Removal-of-batch-effects-using-distribution) (MH, SG)  | [DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1189-z) (DR) | 
| 11.12.2017  | Gosia  | hands-on session #2: mass cytometry  | X | X | 
| 18.12.2017  | Mark  | epigenomics, DNA methylation, ChIP data, gene set analysis  | [Linear models enable powerful differential activity analysis in massively parallel reporter assays](https://www.biorxiv.org/content/early/2017/09/30/196394) (DP, ZY)  |  | 


## Useful Links
[Simply Statistics blog](http://simplystatistics.org/archive/)  
[Getting Genetics Done blog](http://www.gettinggeneticsdone.com/)  
[Omics Omics blog](http://omicsomics.blogspot.ch/)  
[Staying Current in Bioinformatics & Genomics: 2017 Edition](http://www.gettinggeneticsdone.com/2017/02/staying-current-in-bioinformatics-genomics-2017.html)

## Course material

Assuming you have git installed, you can check out the entire set of course materials with the following command (from command line):
```
git clone https://github.com/sta426hs2017/material.git
```  
Alternatively, for a ZIP file of the repository, you can click on the (green) 'Clone or download' (top right) and then click 'Download ZIP'.
